src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.common;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.*;
+
+public class UtilsTest {
+	
+	@Test
+	public void testIsAnyNullTrue(){
```

Comment:
Make sure the method name is according to standards. It should be methodYouTest_ScenaroYouTest_ReturnValue. 

i.e. isAnyNull_emptyList_falseReturned